src/utilities.jl

```diff
@@ -0,0 +1,25 @@
+import Base: ==
+
+function ==(sys1::AbstractSystem, sys2::AbstractSystem)
+    if typeof(sys1) != typeof(sys2)
+        return false
+    end
+    for field in fieldnames(typeof(sys1))
+        if !(getfield(sys1, field) == getfield(sys2, field))
```

Comment:
You did it again :smile:
```suggestion
        if getfield(sys1, field) != getfield(sys2, field)
```

Reply:
I did not change it here because for me it looks more consistent with the `isapprox` method this way and also to directly show that there is a recursion of `==` if a `field` of `sys` is again of type `AbstractSystem`. But I don't have a strong opinion about this.
What is the reason you would change it? More precise code, consistent use of `!=`?

Reply:
I also do not have a strong opinion, but I find it easier to read.
Compare `1 ≠ 2` with `¬(1 = 2)`.